Source/ConversationV1/Models/MessageRequest.swift

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+//Struct for Messages
+public struct MessageRequest: JSONEncodable, JSONDecodable {
+    
+    var input:   [String: JSON]!
+    var context: [String: JSON]?
+    
+    public init(message: String, tags: [String]? = nil, context: [String: JSON]? = nil) {
+        self.input  = ["text" : JSON.String(message)]
+        self.context = context
+    }
+    
+    public init(json: JSON) throws {
```

Comment:
If this class is never initialized from JSON, then you can remove this `init`. If not, then it would be helpful to add a comment here like:

"/// Used internally to initialize a `MessageRequest` model from JSON."